matter_server/server/device_controller.py

```diff
@@ -1510,13 +1518,6 @@ def _on_mdns_service_state_change(
             # We already have a timer to resolve this service, so ignore this callback.
             return
 
-        if service_type == MDNS_TYPE_COMMISSIONABLE_NODE:
-            # process the event with a debounce timer
-            self._mdns_event_timer[name] = self._loop.call_later(
-                0.5, self._on_mdns_commissionable_node_state, name, state_change
-            )
-            return
-
```

Comment:
Can this be handled in a seperate dedicated PR with a breaking change note ?
Because HA is not the only lib consumer of Matter Server. My preference would be to leave it in and not cause breaking changes.

Reply:
> Can this be handled in a seperate dedicated PR with a breaking change note ?

Removed in #1121.

> Because HA is not the only lib consumer of Matter Server. My preference would be to leave it in and not cause breaking changes.

I don't think this influences our server API in any way. We can simply revert the above PR if we actually make use of it.